src/components/Footer.vue

```diff
@@ -8,7 +8,7 @@
         Réseaux
       </h1>
       <div class="flex w-min flex-row">
-        <a class="hover:blur-sm" href="https://www.instagram.com/insalan_officiel">
+        <a class="hover:blur-sm" href="https://www.instagram.com/insalan_officiel" target="_blank">
```

Comment:
There is an error raised by ESlint. You should add `rel="noopener noreferrer"` when using `target="_blank"` for security reason.